gwt-shared-geo-common/src/main/java/nl/aerius/geo/domain/legend/ColorLabelsLegend.java

```diff
@@ -34,6 +34,7 @@ public class ColorLabelsLegend implements Legend {
   private String outlineColor;
   private Integer[] iconSizes;
   private LegendType icon;
+  private Object[] filterables;
```

Comment:
Was this tested? GWT doesn't like `Object` fields... and it seems to kill the startup of Calculator..

Reply:
Locally with codeserver it seemed to work fine. Is this with a deployed version?

Reply:
Yes deployed. My pr built won't start up.